concreate/templates/template.jinja

```diff
@@ -103,17 +103,22 @@ VOLUME ["{{ volume }}"]
 {% endfor %}
 {% endif %}
 
-{%- if workdir %}
+
+{%- if run %}
+{%- if 'workdir' in run %}
 # Specify the working directory
-WORKDIR {{ workdir }}
+WORKDIR {{ run['workdir'] }}
 {% endif %}
 
-USER {{ user }}
+{% if 'user' in run %}
+USER {{ run['user'] }}
+{% endif %}
```

Comment:
> should we still make the run/user instruction optional? This may lead to some confusion where the used user is last user that executed any command.

I think the instruction should remain optional, but should default to root if not specified. There is already an explicit `USER root` earlier on (before packages) in the template. So the default is already root, if no scripts are run. It would be confusing for a `USER` instruction from the scripts section to bleed out to the `CMD` or `ENTRYPOINT` bits at the end. So I'd suggest  adding an `{% else %} USER root` here.